scripts/src/main/resources/scripts/functions

```diff
@@ -741,6 +741,42 @@ function doUnzip() {
   doRunCommand "unzip -qn '${1}' -d '${target_dir}'"
 }
 
+function doInstallWithPackageManager() {
+  pckmngers=("apt-get" "yum" "pacman" "emerge" "zypp" "apk" "dnf" "brew")
+  installString=${1}
```

Comment:
Sorry, it seems I was not clear enough about this feature/story.
The `installString` may differ between packagemanager and therefore this will not properly work for all situations.

My expectation was that I can provide an array of linux/OS specific commands to this function.
Then this function will iterate all of them and always extract the first arg of the command (the packagemanager).
It will check if that command exists and if yes call it prefixed by `sudo`.

Example usage for Python:
```
doInstallWithPackageManager "apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev" "yum -y groupinstall 'Development Tools && yum -y install gcc openssl-devel bzip2-devel libffi-devel" "zypp install automake fdupes gcc gcc-c++ gcc-fortran gdbm-devel gettext-tools gmp-devel intltool libbz2-devel libexpat-devel libffi-devel libnsl-devel lzma-devel make ncurses-devel netcfg openssl-devel pkgconfig readline-devel sqlite-devel xz zlib-devel" 
```

Or simpler case for unzip (you could change doUnzip using this in the same PR):
```
doInstallWithPackageManager "apt-get install unzip" "yum install unzip" "pacman -S unzip" "emerge install unzip" "zypp install unzip" "apk add unzip" "dnf install unzip" "brew install unzip"
```

Reply:
now all fine. Thanks.